src/app/models/game-info.ts

```diff
@@ -45,4 +45,12 @@ export const gameInfo: Record<Game, GameInfo> = {
     modId: 'coi',
     hideColumns: ['beacons', 'pollution', 'power', 'wagons'],
   },
+  [Game.Techtonica]: {
+    icon: 'techtonica',
+    route: 'techtonica',
+    label: 'options.game.techtonica',
+    meta: 'Techtonica',
+    modId: 'tta',
+    hideColumns: [],
```

Comment:
Does this data include information for all columns? I would think we would at _least_ hide the `beacons` column. I haven't inspected the data closely, but you may also want to hide `pollution`, `power`, and/or `wagons`.

Reply:
Fixed